### PR TITLE
Lower difficulty

### PR DESCRIPTION
**Pull request by Jontpan**
I now think this feels good enough to warrant a pull req. So:

* Spawn a second platform on the same y-position if
  * We are at 500 or lower
  * A normal or a trampoline platform has been spawned

If these criteria are met, there is a 60 percent chance to spawn a second platform.

This commit also lowers the scroll speed at the start of the game, and increases this gradually to what we have now. in making this change, the camerafollow became very jumpy if you were faster than the scroll speed, so I changed how the camera follows the player:

* Baseline lerpspeed is lowered (to reduce jumpyness)
* Lerpspeed increases the further away the player is from the center (to always keep the player on the screen)

I think this feels kinda good, what do you think?

#97

### CODE REVIEW

File: 2Dgame/Assets/Scripts/CameraScript.cs [hunk 1]
```diff
@@ -23,12 +23,46 @@ public class CameraScript : MonoBehaviour {
...
+
+    /**
+     * Scrolls up in different speeds, depending on score.
+     * To make the start easier.
+     */
+    void StandardSpeed()
+    {
+        if (transform.position.y < 200)
+        {
+            standardSpeed = 2f;
```
Comment (AxlLind) [inline 2Dgame/Assets/Scripts/CameraScript.cs:48]: It might be just me who's played this game a lot but this speed felt very slow. Maybe change this to 3f? What do you think? Like I would have to jump on a platform several times to even get this speed

File: 2Dgame/Assets/Scripts/CameraScript.cs [hunk 2]
```diff
@@ -23,12 +23,46 @@ public class CameraScript : MonoBehaviour {
...
+        else
+        {
+            standardSpeed = 5.4f;
+        }
+    }
+
+    /**
+     * Lerps faster the farther away from the middle the player is.
+     */
+    float LerpSpeed()
```
Comment (AxlLind) [inline 2Dgame/Assets/Scripts/CameraScript.cs:63]: I really like this change. Made it feel smoother

File: 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs [hunk 3]
```diff
@@ -123,15 +125,44 @@ void choosePlatform()
 
 	/**
 	 * Returns a random x-position to spawn a platform at.
-	 * Makes two platforms in the same position twice in a row
+	 * Makes two platforms in the same position twice
 	 * in a row less likely, though not impossible
 	 */
 	int correctX() {
-		int x = 2 * Random.Range (-3, 4);
+        int x = Random.Range(-6, 7);
```
Comment (AxlLind) [inline 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs]: Why this change? This means 16 different possible x-positions instead of 8. Doesn't 8 places make the game feel more 'blocky' / '8-bity'?

File: 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs [hunk 4]
```diff
@@ -123,15 +125,44 @@ void choosePlatform()
...
+        }
+    }
+
+    /**
+     * Sets the spawnposition for the other platform.
+     * There is always at least two world units between them.
+     */
+    int Xvalue(int otherX)
+    {
+        if (otherX - 6 < -6)
```
Comment (AxlLind) [inline 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs]: So otherX < 0 ?

File: 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs [hunk 5]
```diff
@@ -123,15 +125,44 @@ void choosePlatform()
...
+
+    /**
+     * Spawns a second platform at the same yPostion if 
+     * all the criterias are met.
+     * 
+     * Currently 60 percent chance to spawn a second one.
+     */
+    void SpawnSecond(GameObject platform, float yPosition)
+    {
+        if (Camera.main.transform.position.y < 500 && (platform.Equals(platformPrefabs[0]) || platform.Equals(platformPrefabs[2])) && Random.Range(0, 100) < 60)
```
Comment (AxlLind) [inline 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs]: Is 500 maybe too high to still need "help"?

**Review by AxlLind — COMMENTED** [review]
Nice work! And here I am not doing anything haha

I like this implementation. The 60% chance instead of always dual platforms is really good. I think this will make the game a lot better but we will probably have to have people other than us two test that out.

I feel like the scroll speed in the very beginning is a bit too slow though. Also maybe y at 500 is a bit too high to speed need the game easier?

File: 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs (hunk 3 above)
Comment (Jontpan) [reply 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs]: Don't know, but this change doesn't really belong here either way. Leaving it out of this pull request.

File: 2Dgame/Assets/Scripts/CameraScript.cs (hunk 1 above)
Comment (Jontpan) [reply 2Dgame/Assets/Scripts/CameraScript.cs:48]: I actually don't think this is too slow, as a new player. And, if you don't need it you won't notice it. I don't really think we can make it too easy in the beginning.

File: 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs (hunk 5 above)
Comment (Jontpan) [reply 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs]: Sure, maybe 300? Perhaps the chance of double should decrease gradually.

File: 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs (hunk 4 above)
Comment (Jontpan) [reply 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs]: Shut up.

**Comment by Jontpan** [comment]
Don't sweat it, it's the summer break and I'm really only doing this because I don't have a good enough game to play....

File: 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs (hunk 5 above)
Comment (AxlLind) [reply 2Dgame/Assets/Scripts/PlatformSpawnerScript.cs]: Yeah, that's a good idea

**Review by AxlLind — APPROVED** [review]
So now it's: 

60% at y<250
30% at y<400
0% at larger y?

Looks good! Merge this.
